css/selectors.json

```diff
@@ -272,6 +272,14 @@
     ],
     "status": "standard"
   },
+  ":matches()": {
```

Comment:
This should be [`:matches`](https://developer.mozilla.org/en-US/docs/Web/CSS/:matches).